tests/test_main.py

```diff
@@ -113,6 +113,14 @@ def test_main():
         "--load_experiment_id", "foobar", "--experiment_id", "baz"
     ]
     main()
+    # Run actual main approach, but with option learning.
```

Comment:
wdym by "actual main approach"?

Reply:
I was just copying the language from above in the same file. I can say "NSRT learning" instead